Sources/ValueType.swift

```diff
@@ -49,14 +49,39 @@ extension Int64: ValueType {
 }
 
 extension Array where Element: ValueType {
-    public static func value(from object: Any) throws -> [Element] {
+    public static func value(from object: Any, discardInvalidObjects: Bool = false) throws -> [Element] {
```

Comment:
That could work. I was honestly hoping to find a shorter name, maybe something like `discardErrors:` or `discardFailures`. Whatever is the most intuitive though. I think that should be the #1 criteria